### PR TITLE
Sync with NEST v2.1.1

### PR DESCRIPTION
**Pull request by sophiaandaloro**
Main change in renaming testNEST to execNEST. 
Also had some previous issues in building due to this discrepancy so hoping that this works if I caught all of the testNEST->execNEST changes necessary.

### CODE REVIEW

**Comment by sophiaandaloro** [comment]
Reason for previous build failing: Didn't include new free parameter for skewness in the bindings.cpp which has to unfortunately be hard-coded updated.

**Review by tunnell — APPROVED** [review]

File: HISTORY.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ History
 
 Patch releases mean (the Z number in X.Y.Z version) that the underlying physics has not changed.  Changes to the NEST version will always trigger a minor or major release.  If this library changes such that end users have to change their code, this may also trigger a minor or major release.
 
-NEST v2.1.1
 
-1.3.1 (2020-08-20)
-------------------
+NESTv2.1.1
```
Comment (tunnell) [inline HISTORY.md:8]: Format wrong, this goes below

File: HISTORY.md [hunk 2]
```diff
@@ -37,7 +38,7 @@ Default arguments for GetYields and GetQuanta (see PR #25)
 
 NESTv2.0.1
 
-* testNEST included in nestpy
+* execNEST included in nestpy
```
Comment (tunnell) [inline HISTORY.md:41]: ??

File: HISTORY.md (hunk 1 above)
Comment (tunnell) [reply HISTORY.md:8]: What did this version change?  Please include release notes

File: tests/core_nest_tests.py [hunk 3]
```diff
@@ -169,32 +169,32 @@ def test_nestcalc_get_quanta_defaults(self):
 #                            [0, 1, 2],
 #                            [0, 1, 2])
 
-#class testNESTTest(unittest.TestCase):
+#class execNESTTest(unittest.TestCase):
```
Comment (tunnell) [inline tests/core_nest_tests.py:172]: Can you see if these work?

**Review by tunnell — APPROVED** [review]
Please address comments before merging.

**Pull request merged**